src/teams/teams.controller.ts

```diff
@@ -32,7 +32,7 @@ export class TeamsController {
 
     @ApiOperation({
         summary: "[Roles: Admin] Gets all voyage teams.",
-        description: "[access]: admin <br> For development/admin purpose",
+        description: "For development/admin purpose",
```

Comment:
how come the access info is removed? this is probably going to be an actual endpoint too for the admin dashboard (not just development)